os/arch/arm/src/s5j/s5j_pmstop.c

```diff
@@ -71,8 +71,20 @@
  * Description:
  *   Enter STOP mode.
  *
+ * Input Parameters:
+ *   None
+ *
+ * Returned Value.
+ *   In case of failure a negated errno value is returned to indicate the
+ *   cause of the failure.
+ *
  ****************************************************************************/
-void s5j_pmstop(void)
+int s5j_pmstop(void)
 {
-	s5j_clk_pll_select_mux(false);
+	/* FIXME: implement me */
+	asm("wfi");
```

Comment:
Same as up_cpuidle. What is relationship between them?

Reply:
This is provided by the SOC, but up_cpuidle() is called in case power framework is not supported.